ddtrace/contrib/pynamodb/patch.py

```diff
@@ -0,0 +1,83 @@
+"""
+Trace queries to botocore api done via a pynamodb client
+"""
+
+import os
+from ddtrace.vendor import wrapt
+from ddtrace import config
+import pynamodb.connection.base
+
+
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY, SPAN_MEASURED_KEY
+from ...pin import Pin
+from ...ext import SpanTypes
+from ...utils.formats import deep_getattr, get_env
+from ...utils.wrappers import unwrap
+
+# Pynamodb connection class
+_PynamoDB_client = pynamodb.connection.base.Connection
+
+if "DD_PYNAMODB_SERVICE" in os.environ:
+    service = os.getenv("DD_PYNAMODB_SERVICE")
+else:
+    service = get_env("pynamodb", "service_name")
+
+config._add("pynamodb", {"service_name": service or "pynamodb",})
```

Comment:
```suggestion
config._add("pynamodb", {"service_name": os.getenv("DD_PYNAMODB_SERVICE") or "pynamodb",})
```

We can simplify the service name here. We basically just want to take the environment variable and fall-back to `"pynamodb"` otherwise. So we don't need the stuff on lines 20-23.

`get_env` isn't required as we don't need to backwards support `DATADOG_PYNAMODB_SERVICE` or `{DD,DATADOG}_PYNAMODB_SERVICE_NAME`. This is a mess, I know 🤦

Reply:
Thanks for explaining that! I'll remove them out!